feeds/es.json

```diff
@@ -54,18 +54,6 @@
             "transitland-atlas-id": "f-ezj-informaciónoficial~consorcioregionaldetransportesdemadrid",
             "fix": true
         },
-        {
```

Comment:
Yes, to avoid duplicates. The sources I am using now (from the Spanish Ministry of Transport) provide more reliable and unified feeds. Nearly all the old ones will end up being removed in favour of the ones provided by the ministry.